packages/site-content/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "./lib",
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
+    "module": "CommonJS",
+    "skipLibCheck": false
```

Comment:
false is the default, no?

Reply:
It was set to `true` in the base config. `catalog-server` doesn't compile with it enabled -- something to do with a library called `natural`. I've removed it from the base config and set it to `false` only in `catalog-server`.